lib/msal-browser/src/error/BrowserConfigurationAuthError.ts

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+import { ClientConfigurationError } from "msal-common";
+
+/**
+ * BrowserAuthErrorMessage class containing string constants used by error codes and messages.
+ */
+export const BrowserConfigurationAuthErrorMessage = {
```

Comment:
Same questions here as I left above in `BrowserAuthError`.

Reply:
Not sure I understand this one?

Reply:
will chagne to extend AuthError

Reply:
Done.